tools/cldr-apps/js/src/views/VettingSummary.vue

```diff
@@ -232,6 +232,19 @@ export default {
       return cldrLoad.getLocaleName(locale);
     },
 
+    /**
+     * Display a more user-friendly snapshot id
+     *
+     * @param id like “2022-05-16T08:15:25.083077935Z”
+     * @return like “2022-05-16 08:15 GMT”
+     */
+    humanizeSnapId(id) {
+      if (!id.match(/^\d+\-\d+\-\d+T\d\d:\d\d:\d\d\..+Z$/)) {
+        return id;
+      }
+      return id.replace("T", " ").replace(/:\d\d\..+Z/, " GMT");
```

Comment:
you know, there's this CLDR thing…  😄 

```suggestion
    return new Date(id).toLocaleString();
```

```shell
> new Date('2022-05-16T08:15:25.083077935Z').toLocaleString()
'5/16/2022, 3:15:25 AM'
```

Reply:
that's a good option -- the ticket called for GMT, should we change that?

it will be more robust to keep a fallback (keep id unchanged) if id isn't a date or doesn't match the expected pattern

Reply:
can use the following then (will still use preferred format
```shell
> new Date('2022-05-16T08:15:25.083077935Z').toLocaleString([], {timeZone: 'UTC', timeZoneName: 'short'})
'5/16/2022, 8:15:25 AM UTC'
```

or if you want to lock down the format

```shell
> new Date('2022-05-16T08:15:25.083077935Z').toLocaleString('en-US', {timeZone: 'UTC', timeZoneName: 'short'})
'5/16/2022, 8:15:25 AM UTC'
```